greenpithumb/pi_io.py

```diff
@@ -0,0 +1,43 @@
+#import RPi.GPIO as GPIO
+
+# TODO(mtlynch): Move these pin constants to a configuration file so they're not
+# hardcoded in.
+
+# Raspberry Pi pin to which each component is connected
+PIN_PUMP = 0
+
+
+class IO(object):
+    """Wrapper for input and output on a Raspberry Pi board.
+
+    This wraps the board and allows the caller to read or send signals to
+    the Raspberry Pi's pins.
+    """
+
+    def __init__(self, gpio):
+        """Creates a new input/output wrapper.
+
+        Args:
+            gpio: Raspberry Pi GPIO module.
+        """
+        self._GPIO = gpio
+
+    def turn_pin_on(self, pin):
+        """Turns on a Raspberry Pi GPIO pin.
+
+        Args:
+            pin: Index of Raspberry Pi pin to turn on
+        """
+        self._GPIO.setup(pin, self._GPIO.OUT)
+        self._GPIO.output(pin, 1)
```

Comment:
I think we want `self._GPIO.HIGH` instead of `1` and `self._GPIO.LOW` instead of `0`